images/hubble-proto/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM docker.io/library/golang:1.14.7-alpine3.12 as builder
```

Comment:
any reason for 1.14 but not 1.15 ? I think cilium master now is with golang 1.15.

Reply:
no particular reason, can update to 1.15